policyengine_us/parameters/gov/states/co/tax/income/credits/sales_tax_refund/agi_sources.yaml

```diff
@@ -0,0 +1,16 @@
+description: Colorado accounts for these income sources for the modified adjusted gross income.
+values:
+  2021-01-01:
+    - adjusted_gross_income
+    - tax_exempt_social_security
+    - tax_exempt_interest_income
```

Comment:
This assumes that all tax-exempt interest income is from state and local bonds, which I don't think we want to do per the other conversation.